libs/spark/test/navbar.spec.tsx

```diff
@@ -1,17 +1,16 @@
 import { render } from '@testing-library/react';
 import 'jest-styled-components';
-import { SparkThemeProvider, NavBar, NavbarLink } from '../src';
-import QuestionDuotoneIcon from '@prenda/spark-icons/QuestionDuotone';
+import { SparkThemeProvider, NavBar, NavBarButton } from '../src';
+import QuestionDuotone from '@prenda/spark-icons/QuestionDuotone';
 
 describe('Button', () => {
```

Comment:
Not this PR's fault, but it can fix it

```suggestion
describe('NavBar', () => {
```